csharp/src/SeedLang.Shell/Repl.cs

```diff
@@ -29,6 +29,7 @@ internal sealed class Repl {
     internal Repl(SeedXLanguage language, RunType runType,
                   IEnumerable<VisualizerType> visualizerTypes) {
       _language = language;
```

Comment:
The variable `_language` can be removed and access the language by `_source.Language`.